### PR TITLE
(PUP-5445) Add Arista support

### PR DESCRIPTION
**Pull request by LuvCurves**
This covers work done for QA-2096

### CODE REVIEW

File: acceptance/lib/puppet/acceptance/install_utils.rb [hunk 1]
```diff
@@ -8,11 +8,12 @@ module Acceptance
     module InstallUtils
       PLATFORM_PATTERNS = {
         :redhat        => /fedora|el|centos/,
-        :debian        => /debian|ubuntu/,
+        :debian        => /debian|ubuntu|cumulus/,
         :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
         :solaris_10    => /solaris-10/,
         :solaris_11    => /solaris-11/,
         :windows       => /windows/,
+        :eos           => /eos/,
```
Comment (kylog) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: Paranoid thought: make the regex match more restrictive, e.g. `/eos-4-i386/` or '/^eos-/' or something.

We're right now being bitten by the `el` match on line 10 inadvertently matching `elcapitan`, which has me thinking we should err a bit on the side of restricting matches. E.g., here, what if we start testing against `mirageos` in the future?

Thoughts? /cc @joshcooper @johnduarte

Comment (joshcooper) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: :+1: to more restrictive match, e.g. `beos` would match the regex

File: acceptance/tests/resource/cron/should_be_idempotent.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 test_name "Cron: check idempotency"
-confine :except, :platform => 'windows'
+confine :except, :platform => ['windows']
```
Comment (joshcooper) [inline acceptance/tests/resource/cron/should_be_idempotent.rb]: why the change to windows?

**Comment by LuvCurves** [comment]
Recrafting
